### ISSUE TITLE
Won't load model (A320 neo included w/ msfs)

### ISSUE THREAD

**Issue by alfre375** [bestdani/msfs2blend#23]
[https://drive.google.com/file/d/133q-DjWKLmkjVHlC9RUkhX1uvTy-5q8N/view?usp=sharing](url)
Drive file with screenshot of problem

**Comment by alfre375** [comment]
blender 3.3 is used, but i tried 3.2 and it gave me the same thing

**Comment by Kelleytoons** [comment]
Yeah, I have this issue with no solution.  I think Asobo broke this so you can no longer edit ANY of the Asobo planes.

**Comment by francisjohannes** [comment]
use blender 2.93 it will work for importing the a320neo.

**Comment by Kelleytoons** [comment]
Well, I want to get the Cessna 172 in so unless it works with that I'm not going to try (I've already installed FIVE different Blender versions to get this to work - I have little faith that 2.93 will work).

**Comment by Kelleytoons** [comment]
Okay, I had to prove I was right and I am - 2.93 doesn't work either.

As I said earlier, Asobo broke this.  You can no longer edit the standard models with this plugin.

**Comment by crx97220130101-dotcom** [comment]
如果这个插件因为asobo的原因无法正常导入我该如何在blender中导入A32NX的模型
